warn/scrapers/ky.py

```diff
@@ -0,0 +1,132 @@
+import typing
+from pathlib import Path
+
+import xlrd
+from openpyxl import load_workbook
+
+from .. import utils
+from ..cache import Cache
+
+__authors__ = [
+    "palewire",
+]
+__tags__ = [
+    "excel",
+]
+
+
+def scrape(
+    data_dir: Path = utils.WARN_DATA_DIR,
+    cache_dir: Path = utils.WARN_CACHE_DIR,
+) -> Path:
+    """
+    Scrape data from Kentucky.
+
+    Arguments:
+    output_dir -- the Path were the result will be saved
+
+    Keyword arguments:
+    data_dir -- the Path were the result will be saved (default WARN_DATA_DIR)
+    cache_dir -- the Path where results can be cached (default WARN_CACHE_DIR)
+
+    Returns: the Path where the file is written
+    """
+    # Get the latest workbook
+    cache = Cache(cache_dir)
+    latest_url = "https://kcc.ky.gov/WARN%20notices/WARN%20NOTICES%202022/WARN%20Notice%20Report%2001262022.xls"
+    latest_path = cache.download("ky/latest.xls", latest_url)
+
+    # Open it up
+    latest_workbook = xlrd.open_workbook(latest_path)
+
+    # Loop through the worksheets
+    dict_list = []
+    for sheet in latest_workbook.sheet_names():
+        # Get the data as a list of lists
+        sheet_obj = latest_workbook.sheet_by_name(sheet)
+        sheet_list = parse_xls(sheet_obj)
+
+        # Convert to list of dicts
+        headers = sheet_list[0]
+        for row in sheet_list[1:]:
+            d = {}
+            for x, value in enumerate(row):
+                d[headers[x]] = value
```

Comment:
Could do:
```suggestion
                if "Date" in headers[x] and value:
                    value = xlrd.xldate.xldate_as_datetime(value, latest_workbook.datemode)
                d[headers[x]] = value
```

Result looks like `2017-04-01 00:00:00`, which is consistent with xlsx.

Reply:
Thanks for this tip. I found another way to do in the parser using some native XLRD typing tricks.